win/rl/pynethack.cc

```diff
@@ -33,7 +33,7 @@ checked_conversion(py::object obj, const std::vector<ssize_t> &shape)
 {
     if (obj.is_none())
         return nullptr;
-    auto array = py::array::ensure(obj.release());
+    py::array array = py::array::ensure(obj.release());
```

Comment:
Can you explain why this way is better?

Reply:
I had to look up what `py::array::ensure` actually does and it turns out it returns a `py::array`, which is only 5 more characters than `auto` and would have prevented me wondering what it returns. Hence I changed it.